src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -5278,7 +5278,7 @@ class C
             int exitCode = csc.Run(outWriter);
             Assert.Equal(0, exitCode);
             Assert.Equal(@"
-Microsoft (R) Visual C# Compiler version A.B.C.D
+Microsoft (R) Visual C# Compiler version A.B.C.D (<developer build>)
```

Comment:
>(<developer build>) [](start = 49, length = 19)

I believe hardcoding the string makes the tests fragile. They will break if run against official binary, which we might want to support.